src/ml_models.py

```diff
@@ -726,17 +814,21 @@ def classify(self, x: Array) -> bool:
         assert classification in [True, False]
         return classification
 
-    def _fit(self, X: Array, y: Array) -> None:
```

Comment:
why is it okay to remove these?

Reply:
because not all classes need to normalize, so they might only want to implement `fit` and `predict`